frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1410,7 +1442,11 @@ class Worksheet extends React.Component {
                             // When deleting text, we want the focus to stay at the same index,
                             // unless it is the last item in the worksheet, at which point the
                             // focus goes to the last item in the worksheet.
-                            this.setFocus(items.length === focus ? items.length - 1 : focus, 'end');
+                            this.setFocus(items.length === focus ? items.length - 1 : focus, 0);
+                        }
+                        if (searchDeleted) {
```

Comment:
I'm not quite sure what searchDeleted is supposed to mean, from line 318, it seems to only be checking whether the command was 'rm', why is not called "fromDeleteCommand". Also,the comment says the condition is " no bundle deleted",but  wouldn't it still reach here even if some bundle was deleted?

Reply:
I renamed `searchDeleted` to `searchDirectiveDeleted`, so the naming pattern is the same as `textDeleted`, which represents which items we aim to delete. 
We will execute this if condition only when we delete some bundles through the search directive, since the number of bundles does not change.

Reply:
Have switched the var name to `fromDeleteCommand`